src/Verb.js

```diff
@@ -120,7 +120,7 @@ TinCan client library
 
                 if (this.display === null && typeof _downConvertMap[this.id] !== "undefined") {
                     this.display = {
-                        "und": _downConvertMap[this.id]
+                        "en": _downConvertMap[this.id]
```

Comment:
While we know they are English, I'm not sure this is worth the backwards compatible change. Any client should be accepting "und", so I'd expect this to only negatively effect foreign clients where they don't want to see an English word in a place where they aren't expecting English, except the issue is with 0.9 use cases only and the spec didn't allow for non-English use in that case so they only had one option, that of the English.

Ultimately 0.9 support gets dropped completely in the 1.x roadmap so I'm also inclined to leave this as is until that point and it becomes a non-issue.